spec/models/manageiq/providers/base_manager/refresher_spec.rb

```diff
@@ -52,8 +52,7 @@
         refresher.preprocess_targets
 
         targets_by_ems = refresher.targets_by_ems_id[vm.ext_management_system.id].first
-        expect(targets_by_ems).to be_a(InventoryRefresh::TargetCollection)
-        expect(targets_by_ems.targets.count).to eq(101)
+        expect(targets_by_ems).to be_a(ExtManagementSystem)
```

Comment:
I did have to change this test, but I think the test was actually wrong before.  When you have more than the full_refresh_threshold number of targets you are supposed to have an `ExtManagementSystem` target not a `TargetCollection` with all of the targets.